brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/Alien4CloudFacade.java

```diff
@@ -420,16 +449,16 @@ public Object resolveProperty(String nodeId, Alien4CloudApplication toscaApplica
     }
 
     @Override
-    public Map<String, String> getResolvedAttributes(String nodeId, Alien4CloudApplication toscaApplication) {
-        Map<String, String> resolvedAttributes = MutableMap.of();
+    public Map<String, Optional<?>> getResolvedAttributes(String nodeId, Alien4CloudApplication toscaApplication) {
```

Comment:
What is the benefit of returning `Optional` values over simply not including them in the map. The only caller of this method just discards `absent` values.